Added templates guide and fixed installation guide

sanskarg348 opened the pull request:
- Add description here  
- Mark x inside the square brackets in the checklist below to indicate that you have adhered to the guidelines. 
- Check `Preview` to review your PR. 

------------------------------------------------------------------------
Thank you for your contribution to the documentation of Atri framework. 

Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings. You can check your build by running the following command:
```
$ yarn build
```

- [x] You have checked the spellings and grammar of your text. We recommend using `Code Spell Checker` and `LTeX – LanguageTool grammar/spell checking` extensions in VS Code.

darshitac11 commented:
Please make sure that you always add reference to the issues that you are fixing in your PR.

darshitac11 commented on docs/getting-started/installation/install_with_pipenv.md:
```diff
@@ -57,4 +60,11 @@ If you are using the framework for the first time, we strongly recommend you fol
+- Getting Receiving TypeError upon running atri start ? [view answer](https://heroku.com).
+- Getting Running pipenv install atri results in 'The term 'pipenv' is not recognized as the name of a cmdlet, function, script file, or operable program.' error ? [view answer](https://github.com/Atri-Labs/atrilabs-engine/discussions/338).
+
+Also, if you are still have some unresolved errors you can visit the other questions in the discussions [here,](https://github.com/Atri-Labs/atrilabs-engine/discussions/categories/help-installation-start)
```
grammatical error

darshitac11 reviewed (commented):
@sanskarg348 - I have added my comments. Please check

sanskarg348 commented:
Fixed Issue #34 Now we have clear documentation to use Templates
Fixed Issue #33 Now we have a FAQ page
Fixed Issue #32 Now we have a caution regarding pipenv
